src/DynamoCoreWpf/UI/Themes/Modern/SidebarGridStyleDictionary.xaml

```diff
@@ -8,13 +8,8 @@
     <ControlTemplate x:Key="LibraryThumbTemplate"
                      TargetType="{x:Type Thumb}">
         <Grid>
-            <Rectangle HorizontalAlignment="Stretch"
-                       VerticalAlignment="Stretch"
-                       Width="Auto"
-                       Height="Auto"
-                       Fill="Transparent" />
             <Border x:Name="Rectangle1"
-                    HorizontalAlignment="Stretch"
+                    HorizontalAlignment="Right"
```

Comment:
Remove the margin on the right of the scrollbar.
![image](https://cloud.githubusercontent.com/assets/8158404/6687057/de6a9e06-ccaf-11e4-9a76-6d6187823f1d.png)